client/main.lua

```diff
@@ -97,22 +97,24 @@ CreateThread(function()
                 end
             end
 
-            if canCarjack then
-                local playerid = PlayerId()
-                local aiming, target = GetEntityPlayerIsFreeAimingAt(playerid)
-                if aiming and (target ~= nil and target ~= 0) then
-                    if DoesEntityExist(target) and IsPedInAnyVehicle(target, false) and not IsEntityDead(target) and not IsPedAPlayer(target) then
-                        local targetveh = GetVehiclePedIsIn(target)
-                        for _, veh in ipairs(Config.ImmuneVehicles) do
-                            if GetEntityModel(targetveh) == joaat(veh) then
-                                carIsImmune = true
+            if Config.CarJackEnable then
```

Comment:
This can be amalgamated with the following line to reduce indentation required.
```lua
if canCarjack and Config.CarJackEnable then
```

Reply:
I combined the two and testing

Reply:
Testing completed go ahead test.